Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM public.ecr.aws/docker/library/alpine:3.12
 
 RUN apk --no-cache --update add bash git \
+    jq curl \
```

Comment:
required to reduce the amount of api calls
as well to pass in the Authorization header to have higher rate limits.